swagger-templates/src/main/java/com/okta/swagger/codegen/AbstractOktaJavaClientCodegen.java

```diff
@@ -415,6 +423,11 @@ else if ("read".equals(alias) || "create".equals(alias)) {
                                 }
                             });
 
+                            //do not implement interface Deletable when delete method has some arguments
```

Comment:
Why so? Can you add some code comments explaining why?

Reply:
`Deletable` defined as public interface with only one method `void delete();` and implementable only for objects with CRUD operation `delete()`

But for objects like `AuthorizationServerPolicy`  `delete` method requere 2 args: `authServerId` and `policyId`. In this case `policyId` will be taken from `src: id` but `authServerId` should be passed manually, so this object is not self-deletable and should not implement the `Deletable` interface